testing/test_pytest_html.py

```diff
@@ -140,7 +140,9 @@ def test_xpass(): pass
 
     def test_setup_error(self, testdir):
         testdir.makepyfile("""
-            def pytest_funcarg__arg(request):
```

Comment:
Funcarg is deprecated since [Pytest 4](https://github.com/pytest-dev/pytest/issues/1684).